raft/src/main/java/org/apache/kafka/raft/internals/RecordsIterator.java

```diff
@@ -49,17 +50,20 @@
     // Number of bytes from records read up to now
     private int bytesRead = 0;
     private boolean isClosed = false;
+    private boolean doCrcValidation = false;
```

Comment:
This should be `private final boolean doCrcValidation;`. It should be moved with the rest of the `final` fields.